### PR TITLE
cmake: limit dll name to 8 characters for os/2.

### PR DESCRIPTION
**Pull request by sezero**

### CODE REVIEW

**Comment by evpobr** [comment]
Hi @sezero !

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -50,12 +50,17 @@ if(BUILD_SHARED_LIBS AND WIN32)
...
-  endif()
+    endif()
   else()
     if(MINGW OR CYGWIN)
       set_target_properties(samplerate PROPERTIES RUNTIME_OUTPUT_NAME "samplerate-${libsamplerate_VERSION_MAJOR}")
     endif()
   endif()
+  if(OS2)
+    # OS/2 doesn't support a DLL name longer than 8 characters.
+    set_target_properties(samplerate PROPERTIES OUTPUT_NAME "SAMPRATE")
```
Comment (evpobr) [inline src/CMakeLists.txt:61]: Should it be `RUNTIME_OUTPUT_NAME`?

Comment (sezero) [reply src/CMakeLists.txt:61]: I'm not much of a cmake guy:  if the effect is the same, i.e. the generated dll will be named SAMPRATE.dll, then I'll make the change and force-push.

Comment (evpobr) [reply src/CMakeLists.txt:61]: Ok, let's  leave as it is. I don't know how it is more correct here, this is not a very popular operating system.

**Comment by evpobr** [comment]
Thanks for help!

**Pull request merged**